v5/README.md

```diff
@@ -0,0 +1,83 @@
+# Adyen [online payment](https://docs.adyen.com/online-payments) integration demos
+
+This folder contains the previous version of the sample application that uses Adyen Web 5.68.x.
```

Comment:
Better to make it bold and put it on top, as we want to make sure that this is an archive of an older version, let's recommend the latest versions of Adyen.Web Drop-in
> [!] This folder contains...